src/Resources/config/grids/setono_sylius_analytics_admin_property.yaml

```diff
@@ -9,6 +9,11 @@ sylius_grid:
                 trackingId:
                     type: string
                     label: setono_sylius_analytics.ui.tracking_id
+                enabled:
+                    type: twig
+                    label: setono_sylius_analytics.ui.enabled
+                    options:
+                        template: "@SetonoSyliusAnalyticsPlugin/Admin/Grid/Field/enabled.html.twig"
```

Comment:
I think there is a Sylius template for this. Can you find that and use that? :)

Reply:
yes give me a moment